apps/pie-storybook/package.json

```diff
@@ -23,7 +23,7 @@
     "@justeattakeaway/pie-link": "0.11.0",
     "@justeattakeaway/pie-modal": "0.30.0",
     "@justeattakeaway/pie-notification": "0.0.0",
-    "@justeattakeaway/pie-toggle-switch": "0.15.0",
+    "@justeattakeaway/pie-switch": "0.0.0",
```

Comment:
I'm not sure what this will do to the component changelog – it might look a bit strange though.

As it's only a prerelease version, I'd likely leave this to be v0.15, as then the changelog will be correct in terms of describing the changes.